release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3058,6 +3058,9 @@ In the following tables, features are marked with the following statuses:
 // HCIDOCS-365 - This issue should be fixed soon, in a 4.16.z release
 * Do not update firmware for the `BareMetalHosts` (BMH) resource by editing the `HostFirmwareComponents` resource. Otherwise, the BMH remains in the `Preparing` state and executes the firmware update repeatedly. There is no workaround. (link:https://issues.redhat.com/browse/OCPBUGS-35559[*OCPBUGS-35559*])
 
+// HCIDOCS-371
+* Deploying an installer-provisioned cluster on bare metal fails when a proxy is used. A service in the bootstrap virtual machine cannot access IP address `0.0.0.0` through the proxy because of a regression bug. As a workaround, add `0.0.0.0` to the `noProxy` list. See xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#ipi-install-setting-proxy-settings-within-install-config_ipi-install-installation-workflow[Setting proxy settings] for details. (link:https://issues.redhat.com/browse/OCPBUGS-35818[*OCPBUGS-35818*])
```

Comment:
We usually prefer the wording starting with "For more information, ..."

```suggestion
* Deploying an installer-provisioned cluster on bare metal fails when a proxy is used. A service in the bootstrap virtual machine cannot access IP address `0.0.0.0` through the proxy because of a regression bug. As a workaround, add `0.0.0.0` to the `noProxy` list. For more information, see xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#ipi-install-setting-proxy-settings-within-install-config_ipi-install-installation-workflow[Setting proxy settings]. (link:https://issues.redhat.com/browse/OCPBUGS-35818[*OCPBUGS-35818*])
```